worker/src/get_worker.rs

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I Suggest moving the `get_worker.rs` and the `get_tokio_handle.rs` to a module `globals` and rename the files to `tokio_handle.rs` and `worker.rs`

Reply:
Done. 
Although imo `globals` is not the best name, since that only describes the implementation. The trait itself is an accessor to certain components, which is its primary purpose. The fact that its implementation uses a global variable is only secondary.

Reply:
I agree about the suboptimal name, but I don't have a better one yet. :)